paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -38,6 +39,29 @@ internal data class PaymentMethodMetadata(
     val hasCustomerConfiguration: Boolean,
     val financialConnectionsAvailable: Boolean = DefaultIsFinancialConnectionsAvailable(),
 ) : Parcelable {
+    constructor(
```

Comment:
Should we make this a factory method instead of a constructor? In practice I don't think it's much different, but conceptually it seems like this should be a factory `create` method.